### PR TITLE
Bug 1187645 - Refactored check for invalid slug characters.

### PR DESCRIPTION
**Pull request by jezdez**
That includes adding the percentage sign to it.

### CODE REVIEW

**Comment by groovecoder** [comment]
Code looks good. The bug seems to really want a regression test to make sure we don't re-introduce this later?

**Comment by jezdez** [comment]
Agreed! Test forthcoming

**Comment by jezdez** [comment]
@groovecoder Done

File: kuma/wiki/tests/test_views.py [hunk 1]
```diff
@@ -1361,9 +1358,15 @@ def _createAndRunTests(slug):
             locale = settings.WIKI_DEFAULT_LANGUAGE
             foreign_locale = 'es'
             new_doc_url = reverse('wiki.new_document')
-            invalid_slug = invalid_slug1 = "some/thing"
-            invalid_slug2 = "some?thing"
-            invalid_slug3 = "some thing"
+            invalid_slug = "some/thing"
```
Comment (groovecoder) [inline kuma/wiki/tests/test_views.py]: Do we still need this `invalid_slug` varaible? Or can we use `invalid_slugs[0]` below?

Comment (jezdez) [inline kuma/wiki/tests/test_views.py]: Ah, hm, not sure

**Pull request merged**